### ISSUE TITLE
Nested field selector in collection.list lose the value when you edit all items

### ISSUE THREAD

**Issue by adi8i** [gantry/gantry5#1817]
Whit nested field selector I mean `.someting.something`, below are two screens. The value is there when you edit and individual item, but it's lost when you edit all items.

Here's the code, if you want to test it. Just in case if it will be asked, `icon.icon` doesn't make any difference, it can be anything else.
```
name: AI Test
description:
type: particle

form:
  fields:
    enabled:
      type: input.checkbox
      label: Enabled
      description: Globally enable the particle.
      default: true

    items:
      type: collection.list
      label: Items 
      overridable: false
      array: true 
      ajax: true 
      value: name 
      fields:

        .icon.icon:
          type: input.icon 
          label: Nested Filed 
        .another_icon:
          type: input.icon 
          label: Simple Field
```

![1](https://cloud.githubusercontent.com/assets/9744973/22176780/235708cc-e01a-11e6-8b88-e6197b3abdeb.PNG)
![2](https://cloud.githubusercontent.com/assets/9744973/22176781/23ebeb5e-e01a-11e6-9087-153f91dc0bf6.PNG)

**Comment by mahagr** [comment]
I am able to reproduce the issue.

**Comment by adi8i** [comment]
It may be back in 5.4.7

**Comment by mahagr** [comment]
Please create a new issue if you can reproduce the issue.

**Comment by joomlabeat** [comment]
@mahagr 

I arrived here because I am getting a similar issue.

I refactored the yaml of a particle. The particle contains collections. So, foreach item in the collection, I re-organised its fields within tabs.

Now, when I open to edit all items - all fields lose their values.

*This is a Joomla site.
I checked about the version of gantry on that site. I found out that it was running gantry 5.5.7. 
But no update was appearing in the update manager. Also, there was nothing in the Update sites for Gantry - even after "rebuilding" the sites' list. 
I updated Gantry manually to 5.5.15 - The issue with the Edit All Items in the particle still exists.
